proto/cosmos/staking/v1beta1/authz.proto

```diff
@@ -19,7 +19,7 @@ message StakeAuthorization {
   // empty, there is no spend limit and any amount of coins can be delegated.
   cosmos.base.v1beta1.Coin max_tokens = 1 [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coin"];
   // validators is the oneof that represents either allow_list or deny_list
-  oneof validators {
+  oneof validators_list {
```

Comment:
Will it break nicely for clients?
What happens in v0.51 if you have a marshaled authorization from v0.50?
I thought we shouldn't be proto breaking.

Reply:
the byte representation is the same, this seems like a breaking change that would be for the better as well. Two validators in a single folder is confusing.

Reply:
Right, the value is still the same, but the field name isn't.
So it could be quite breaking if someone depend on the marshalled authorization.
Not sure if it is even a use case, but that is still something to consider, imho.

Reply:
i think proto bytes repr is the same, what is different is the JSON repr. 

When would this break:
- it would break replayability on TXs that relied on JSON signing (amino?).